components/hero/readme.md

```diff
@@ -1,6 +1,6 @@
-**This component is deprecated.** You can use the [Call-out component](call-out)
-in place of a centered Hero, or the [Split component](split)
-in place of a Hero with image.
+**This component is deprecated.** You can use the [Display component](display)
```

Comment:
Think this should be changed to something else?

Reply:
Doh, good catch... that's a leftover from when I couldn't think of a name.